src/pages/App.tsx

```diff
@@ -75,6 +76,7 @@ export default function App(): ReactElement {
                   ))}
                   <Redirect from="/pools/:route/:action" to="/pools" />
                   <Route exact path="/risk" component={Risk} />
+                  <Route exact path="/vesting-claim" component={VestingClaim} />
```

Comment:
optimization: Could you make Vesting (and Risk, while you're at it) a lazy-loaded route? Should be a 1 line change 
https://reactjs.org/docs/code-splitting.html#route-based-code-splitting